hybrid/hybrid_simulation.py

```diff
@@ -21,6 +21,7 @@
 from hybrid.layout.hybrid_layout import HybridLayout
 from hybrid.dispatch.hybrid_dispatch_builder_solver import HybridDispatchBuilderSolver
 from hybrid.log import hybrid_logger as logger
+from hybrid.controls import controls_analysis
```

Comment:
I don't know if I am a fan of the way this is being integrated into `hybrid_simulation`...  Correct me if I am wrong but it seems like `controls_analysis` is just a way to pass information from outside of `hybrid_simulation` to the dispatch and then do some quick calculations and print some results. Passing information could be done through `dispatch_options`. The quick calculations and printing results could live in the `Grid` class. I am open to other ideas

Reply:
I agree that the inputs to the new dispatch should be part of dispatch options, and that hybrid.controls should not be part of hybrid_simulation as they are more output metrics / analysis than fundamental to performing the hybrid simulation.